tardis/simulation/base.py

```diff
@@ -415,14 +416,16 @@ def log_plasma_state(
 
         plasma_state_log = pd.DataFrame(
             index=np.arange(len(t_rad)),
-            columns=["t_rad", "next_t_rad", "w", "next_w"],
+            columns=["t_rad (K)", "next_t_rad (K)", "w", "next_w"],
```

Comment:
why are the column names changed?

Reply:
The columns name had an addition of the Unit. The radiative temperature has a specified Unit next to it ie K.

Reply:
Column names have been reverted back to original values.